conformance/lang/expressions/unary-expression/unary_minus_expression.balt

```diff
@@ -0,0 +1,1078 @@
+    io:println(b == -5); // @output true
+}
+
+Test-Case: output
+Description: Test unary expression as the operand of unary negation.
+Labels: DecimalNumber, int, unary-minus, unary-plus, unary-complement
+
+function init() {
+    int a = 5;
+    a = -+5;
+    io:println(a); // @output -5
+
+    a = 5;
+    a = --5;
+    io:println(a); // @output 5
+
+    a = 5;
+    a = -~5;
+    io:println(a); // @output 6
+}
+
+Test-Case: output
+Description: Test type cast expression in integer unary minus operands.
+Labels: DecimalNumber, int, type-cast-expr, unary-minus
+
+function init() {
+    int i = -<int> 6;
+    io:println(i); // @output -6
+}
+
+Test-Case: output
+Description: Test type cast expression in float unary minus operands.
+Labels: DecimalNumber, float, type-cast-expr, unary-minus
+
+function init() {
+    float i = -<float> 6;
+    io:println(i); // @output -6.0
+}
+
+Test-Case: output
+Description: Test shift expressions as unary negative operands.
+Labels: DecimalNumber, int, shift-expr, unary-minus
+
+function init() {
+    int p = -(1 << 2);
+    io:println(p); // @output -4
+}
+
+Test-Case: output
+Description: Test binary bitwise expressions as unary negative operands.
+Labels: binary-bitwise-expr, DecimalNumber, int, unary-minus
+
+function init() {
+    int a = -(2 & 6);
+    int b = -(2 ^ 6);
+    io:println(a); // @output -2
+    io:println(b); // @output -4
+}
+
+Test-Case: output
+Description: Test conditional expressions as unary negative operands.
+Labels: binary-conditional-expr, DecimalNumber, error, int, ternary-conditional-expr, union-type, unary-minus, optional-type, nil-literal
+
+function init() {
+    int|error a = 12;
+    int? b = ();
+    int c = -(a is error ? 2 : a);
+    int d =  -(b ?: 13);
+    io:println(c); // @output -12
+    io:println(d); // @output -13
+}
+
+Test-Case: output
+Description: Test checking expressions as unary negative operands.
+Labels: check, checkpanic, DecimalNumber, error, int, unary-minus, union-type
+
+function init() returns error? {
+    int|error a = 12;
+    int c = -check a;
+    int d = -checkpanic a;
+    io:println(c); // @output -12
+    io:println(d); // @output -12
+}
+
+Test-Case: output
+Description: Test static type of result is contextually expected type.
+Labels: DecimalNumber, decimal, float, int, unary-minus
+
+function init() {
+    float a = -12;
+    decimal b = -12;
+    io:println(a); // @output -12.0
+    io:println(b); // @output -12
+}
+
+Test-Case: error
+Description: Test the static type of operands belonging to non-numeric basic types resulting in an error
+            in unary expression.
+Labels: string, unary-minus
+
+function errorFunction(string a) {
+    string _ = -a; // @error - is not allowed with operands of non-numeric basic types
+}
+
+Test-Case: error
+Description: Test for int the static type of the result being the basic type of the operands via invalid assignment
+             with inference, where operands are of numeric types.
```

Comment:
"Test for int the static type ..." does not make a correct sentence. 

How about,
"Test the static type of the result is the basic type of the static type of the operand, for int type operands via invalid assignment with inference"